cmd/acb/build_test.go

```diff
@@ -12,14 +12,13 @@ import (
 
 func TestCreateBuildTask(t *testing.T) {
 	buildCmd := &buildCmd{
-		dockerfile:   "HelloWorld/Dockerfile",
-		registryUser: "user",
-		registryPw:   "pw",
-		context:      "src",
-		tags:         []string{"foo:latest", "bar/qux"},
-		pull:         true,
-		noCache:      false,
-		dryRun:       true,
+		dockerfile:  "HelloWorld/Dockerfile",
+		credentials: []string{"foo.azurecr.io;user;pw"},
```

Comment:
you can add a test here to ensure that the buildCmd's specified creds == the generated task's creds

Reply:
we do have that in `credential_test.go`. Is that the one you're talking about?

Reply:
done